### PR TITLE
IGNITE-23634 Make hardcoded REBALANCE_RETRY_DELAY_MS configurable

### PR DESCRIPTION
**Pull request by kgusakov**
https://issues.apache.org/jira/browse/IGNITE-23634

### CODE REVIEW

File: modules/table/src/integrationTest/java/org/apache/ignite/internal/table/distributed/disaster/ItHighAvailablePartitionsRecoveryTest.java [hunk 1]
```diff
@@ -82,6 +88,21 @@ void testHaRecoveryOfTwoZones() throws InterruptedException {
         waitAndAssertStableAssignmentsOfPartitionEqualTo(node, table22, PARTITION_IDS, Set.of(node.name()));
     }
 
+    @Test
+    void testInvaliPartitionResetTimeoutUpdate() {
```
Comment (alievmirza) [inline modules/table/src/integrationTest/java/org/apache/ignite/internal/table/distributed/disaster/ItHighAvailablePartitionsRecoveryTest.java:92]: typo

File: modules/configuration-system/src/main/java/org/apache/ignite/internal/configuration/utils/SystemDistributedConfigurationPropertyHolder.java [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+            updateSystemProperties(ctx.newValue(), ctx.storageRevision());
+
+            return CompletableFuture.completedFuture(null);
+        });
+    }
+
+    /**
+     * Init property value, but doesn't call the listener.
+     */
+    public void init() {
```
Comment (alievmirza) [inline modules/configuration-system/src/main/java/org/apache/ignite/internal/configuration/utils/SystemDistributedConfigurationPropertyHolder.java:80]: please add a bit more details why we need this init and when it must be called

**Review by alievmirza — APPROVED** [review]

File: modules/table/src/integrationTest/java/org/apache/ignite/internal/table/distributed/disaster/ItHighAvailablePartitionsRecoveryTest.java (hunk 1 above)
Comment (kgusakov) [reply modules/table/src/integrationTest/java/org/apache/ignite/internal/table/distributed/disaster/ItHighAvailablePartitionsRecoveryTest.java]: Fixed

File: modules/configuration-system/src/main/java/org/apache/ignite/internal/configuration/utils/SystemDistributedConfigurationPropertyHolder.java (hunk 2 above)
Comment (kgusakov) [reply modules/configuration-system/src/main/java/org/apache/ignite/internal/configuration/utils/SystemDistributedConfigurationPropertyHolder.java:82]: Added.

**Pull request merged**